about.html

```diff
@@ -0,0 +1,105 @@
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8">
+        <link rel="stylesheet" href="stetics.css"> 
+        <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Meera+Inimai&display=swap" rel="stylesheet">
+        <link rel="shortcut icon" href="img/github2.png" type="image/x-icon">
+    </head>
+    <body>
+        <nav>
+            <img class="socialmediaicon" src="img/facebookicon.png" alt="facebookicon">
+            <img class="socialmediaicon" src="img/twittericon.png" alt="twittericon">
+            <div>English</div>
+            <div>My Page</div>
+            <div>Logout</div>
+        </nav>
+        <header class="dflex spacebetween">
+            <img class="menu" src="img/Threebars.png" alt="menu">
+            <div class="menudesktop">
+                <ul class="navbarlist dflex">
+                    <li><a href="about.html">About</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                    <li class="campaign">Stand up Campaign</li>
+                </ul>
+            </div>
+            <a class="logo" href="index.html"><img class="logoimgmobile" src="img/microphone-14646.png" alt="logo"><img class="logoimgdesktop" src="img/black-microphone-64px.png" alt="logo"></a>
```

Comment:
Great job @EmiLoKiJu 🤝

- [ ] [Optional] Kindly ensure your mobile design for both the Home and About sections doesn't have this image but rather sticks to the design provided.
- Your design
![your-mobile](https://user-images.githubusercontent.com/63917725/233514997-4bc08094-4d02-45c2-ad9a-2325a15210d8.PNG)
- Given design
![given-mobile](https://user-images.githubusercontent.com/63917725/233515014-38804267-1601-4f06-a71e-784b95c531f5.PNG)